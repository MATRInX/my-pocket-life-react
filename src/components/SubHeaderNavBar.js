import React from 'react';
import { Link } from 'react-router-dom';
import LINKS from '../fixtures/links';

const SubHeaderNavBar = () => (
  <ul>
    <li>
      <Link to={LINKS.POCKET_SCRAPBOOKING}>
        Pocket Scrapbooking
      </Link>
    </li>
    <li>
      <Link to={LINKS.INSPIRATION}>
        Inspiracje
      </Link>
    </li>
    <li>
      <Link to={LINKS.DIY}>
        DIY
      </Link>
    </li>
    <li>
      <Link to={LINKS.REVIEWS}>
        Recenzje
      </Link>
    </li>
    <li>
      <Link to={LINKS.COMMUNITY}>
        Społeczność
      </Link>
    </li>
  </ul>
);

export default SubHeaderNavBar;