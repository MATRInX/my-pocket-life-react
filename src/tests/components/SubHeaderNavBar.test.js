import React from 'react';
import { shallow } from 'enzyme';
import SubHeaderNavBar from '../../components/SubHeaderNavBar';

test('should correctly render SubHeaderNavBar', () => {
  const wrapper = shallow(<SubHeaderNavBar />);
  expect(wrapper).toMatchSnapshot();
});