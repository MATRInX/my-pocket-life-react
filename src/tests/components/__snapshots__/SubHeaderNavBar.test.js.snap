// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render SubHeaderNavBar 1`] = `
<ul>
  <li
    onClick={[Function]}
  >
    <Link
      replace={false}
      to="/kategoria/pocket-scrapbooking"
    >
      Pocket scrapbooking
    </Link>
  </li>
  <li
    onClick={[Function]}
  >
    <Link
      replace={false}
      to="/kategoria/inspiracje"
    >
      Inspiracje
    </Link>
  </li>
  <li
    onClick={[Function]}
  >
    <Link
      replace={false}
      to="/kategoria/diy"
    >
      DIY
    </Link>
  </li>
  <li
    onClick={[Function]}
  >
    <Link
      replace={false}
      to="/kategoria/recenzje"
    >
      Recenzje
    </Link>
  </li>
  <li
    onClick={[Function]}
  >
    <Link
      replace={false}
      to="/kategoria/spolecznosc"
    >
      Społeczność
    </Link>
  </li>
</ul>
`;
